functions/submission-created.js

```diff
@@ -0,0 +1 @@
+!function(e,t){for(var n in t)e[n]=t[n]}(exports,function(e){var t={};function n(r){if(t[r])return t[r].exports;var o=t[r]={i:r,l:!1,exports:{}};return e[r].call(o.exports,o,o.exports,n),o.l=!0,o.exports}return n.m=e,n.c=t,n.d=function(e,t,r){n.o(e,t)||Object.defineProperty(e,t,{enumerable:!0,get:r})},n.r=function(e){"undefined"!=typeof Symbol&&Symbol.toStringTag&&Object.defineProperty(e,Symbol.toStringTag,{value:"Module"}),Object.defineProperty(e,"__esModule",{value:!0})},n.t=function(e,t){if(1&t&&(e=n(e)),8&t)return e;if(4&t&&"object"==typeof e&&e&&e.__esModule)return e;var r=Object.create(null);if(n.r(r),Object.defineProperty(r,"default",{enumerable:!0,value:e}),2&t&&"string"!=typeof e)for(var o in e)n.d(r,o,function(t){return e[t]}.bind(null,o));return r},n.n=function(e){var t=e&&e.__esModule?function(){return e.default}:function(){return e};return n.d(t,"a",t),t},n.o=function(e,t){return Object.prototype.hasOwnProperty.call(e,t)},n.p="",n(n.s=0)}([function(e,t){const n="ideas42ventures",r={"Access-Control-Allow-Origin":""+n,"Access-Control-Allow-Headers":"Content-Type","Access-Control-Allow-Methods":"POST"};function o(e,t,n){e(null,{statusCode:t,headers:r,body:JSON.stringify({msg:n})})}t.handler=function(e,t,r){!function(e,t){const{host:r}=t.multiValueHeaders,u=r[0];if("POST"!==t.httpMethod)return o(e,400,"Method not allowed");if(!u.include(n))o(e,400,"You are not allowed to submit an email address from here")}(r,e);const u=JSON.parse(e.body).payload.email;return/^[^@]+@[^@]+\.[^@]+$/.test(u)?o(r,200,"Thanks for signing up"):o(r,400,"This is an invalid email")}}]));
```

Comment:
Do we need to commit the compiled functions to the repo? Usually that's not what we want.

Reply:
Not at all, fixed!